assets/css/initial-setup/main.styl

```diff
@@ -6,6 +6,17 @@
   margin: 30px auto;
 }
 
+.modal-open {
+
+  .modal{
```

Comment:
Are you going to run the code _beautifier_  now or in a separate PR?

Reply:
There is no repo-wide beautifier yet. Coming in the next release.